src/control/nav2_dwa_controller/src/dwa_controller.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+#include <chrono>
+#include "nav2_dwa_controller/dwa_controller.hpp"
+
+using namespace std::chrono_literals;
+using nav2_tasks::TaskStatus;
+
+namespace nav2_dwa_controller
+{
+
+DwaController::DwaController(const std::string & name)
+: nav2_tasks::FollowPathTaskServer(name)
+{
+  RCLCPP_INFO(get_logger(), "DwaController::DwaController");
+}
+
+DwaController::~DwaController()
+{
+  RCLCPP_INFO(get_logger(), "DwaController::~DwaController");
+}
+
+TaskStatus
+DwaController::execute(const nav2_tasks::FollowPathCommand::SharedPtr /*command*/)
+{
+  RCLCPP_INFO(get_logger(), "DwaController::execute");
+
+  // Spin here for a bit to fake out some processing time
```

Comment:
@crdelsey - are we filing an issue here to track that DWA Controller is only a stub right now?